deps/install.sh

```diff
@@ -42,6 +42,7 @@ build_and_install() {
 
 ## If the path was not given, ask for prior consent about installation to system path.
 if [ -n "$1" ]; then
+    set -- $(mkdir -p $1 && cd $_ && pwd)
```

Comment:
```suggestion
    set -- $(mkdir -p $1 && cd $1 && pwd)
```
둘의 동작에 차이가 없다면 `$_`보다 `$1` 사용하는 편이 읽기 쉽습니다.